src/app/charts/row-chart/row-chart-view/row-chart-view.component.ts

```diff
@@ -235,12 +235,10 @@ export class RowChartViewComponent implements OnInit, OnDestroy, AfterViewInit,
                     }
                 })
                 .on('renderlet', function(chart) {
-                    if (self.firstRender) {
```

Comment:
firstRender is initialized to true, and set to false here, but is it actually used/referenced anywhere?

Reply:
Thank you, I removed the unused code